README.rst

```diff
@@ -14,23 +14,32 @@
    :target: https://pypi.python.org/pypi/openupgradelib
    :alt: Pypi Package
 
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl-3.0
+   :alt: License: AGPL-3
+
 ===============================
 OpenUpgrade Library
 ===============================
 
 A library with support functions to be called from Odoo migration scripts.
+For information on how to develop and contribute
 
 * Free software: AGPL-3 license
 * Documentation: https://oca.github.io/openupgradelib
 
 Install
 -------
 
-Always get the latest version through:
+Always get the latest version through either pip or pip3:
 
-pip/pip3 install --ignore-installed git+https://github.com/OCA/openupgradelib.git@master
+``pip install --ignore-installed git+https://github.com/OCA/openupgradelib.git@master``
```

Comment:
Now that the release to Pypi process is automated, the library is released regularly and there is no need to install from Github anymore.

Reply:
lol ffs, try and do something little 3 years ago. Will fix Wednesday.